doc/nrf/known_issues.rst

```diff
@@ -871,7 +878,7 @@ KRKNWK-14180: The QSPI sleep mode is not handled efficiently in Matter samples o
   QSPI is active during every Bluetooth LE connection in the Matter samples that are programmed on the nRF53 SoC.
   This results in higher power consumption, for example during commissioning into the Matter network.
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
```

Comment:
Could we still add one more Matter issue?
```
KRKNWK-15088: Android CHIP Tool freeze on changing the sensor type
  When a user changes the current sensor type having activated the monitoring of another sensor type, the application freezes.
  **Workaround:** Kill and restart the application. Then select the desired sensor type again.
```

Reply:
Done, please re-check @Damian-Nordic

Reply:
Thank you!